viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -849,15 +849,21 @@ export class Cognite3DViewer {
 
   /**
    * Sets the color used as the clear color of the renderer.
-   * @param color
+   * @param backgroundColor
+   * @param backgroundColor.color
+   * @param backgroundColor.alpha
    */
-  setBackgroundColor(color: THREE.Color): void {
+  setBackgroundColor(backgroundColor: { color?: THREE.Color; alpha?: number }): void {
     if (this.isDisposed) {
       return;
     }
 
-    this.renderer.setClearColor(color);
-    this.spinner.updateBackgroundColor(color);
+    if (!backgroundColor.color) {
+      backgroundColor.color = this.renderer.getClearColor(new THREE.Color());
+    }
+
+    this.renderer.setClearColor(backgroundColor.color, backgroundColor.alpha);
```

Comment:
Small issue here. If you only set color, the previous alpha value is overwritten.

Concider this notation (which will also resolve the issue):

```suggestion
    const color = backgroundColor.color ?? this.renderer.getClearColor(new THREE.Color());
    const alpha = backgroundColor.alpha ??  this.renderer.getClearAlpha();

    this.renderer.setClearColor(color, alpha);
```